src/cdk/drag-drop/resets.scss

```diff
@@ -0,0 +1,10 @@
+@layer cdk-resets {
+  .cdk-drag-preview {
+    background: none;
+    border: none;
+    pointer-events: none;
```

Comment:
I see `pointer-events` and `margin` are set both here and in the strictly-enforced styles styles, do we want them to be user overridable or no?

Reply:
Removed.